common/views/components/Dot/Dot.tsx

```diff
@@ -16,7 +17,7 @@ type Props = {
 };
 
 const Dot: FunctionComponent<Props> = ({ color }: Props) => {
-  return <DotEl className={`font-${color}`} />;
+  return <DotEl className={classNames({ [`font-${color}`]: true })} />;
```

Comment:
Why do we need `classNames()` here?

Reply:
I left a comment just above, I went through the doc and explained as much as I could for unclear things

Reply:
It's not ideal but one day we'll get rid of font-[color] 👀